mlflow/johnsnowlabs/__init__.py

```diff
@@ -288,27 +287,30 @@ def log_model(
     run_id = mlflow.tracking.fluent._get_or_start_run().info.run_id
     run_root_artifact_uri = mlflow.get_artifact_uri()
     remote_model_path = None
-    if _should_use_mlflowdbfs(run_root_artifact_uri):
-        remote_model_path = append_to_uri_path(
-            run_root_artifact_uri, artifact_path, _JOHNSNOWLABS_MODEL_PATH_SUB
-        )
-        mlflowdbfs_path = _mlflowdbfs_path(run_id, artifact_path)
-        with databricks_utils.MlflowCredentialContext(
-            get_databricks_profile_uri_from_artifact_uri(run_root_artifact_uri)
-        ):
-            try:
-                _unpack_and_save_model(spark_model, mlflowdbfs_path)
 
-            except Exception as e:
-                raise MlflowException("failed to save johnsnowlabs model via mlflowdbfs") from e
+    # commented for now because spark flavor is mlflowdbfs it, but it has many issues on databricks
```

Comment:
I think you could remove it directly. You could add it back if you need in the future

Reply:
Sure removed it